### ISSUE TITLE
improve suite/runsuite.cmake: label known failures; ssh support; results in review text

### ISSUE THREAD

**Issue by derekbruening** [DynamoRIO/dynamorio#121]
_From [derek.br...@gmail.com](https://code.google.com/u/117968039472581148324/) on April 13, 2009 00:22:22_

lower-priority nice-to-have features to augment suite/runsuite.cmake:
- have a list of known failures and label w/ " (known: i#XX)"
- ssh support like old runregression
- Makefile target
- replace package.\* with ctest -S script?
- hook up results to codereview.cmake?
- run tests in parallel: already covered by issue #111

_Original issue: http://code.google.com/p/dynamorio/issues/detail?id=121_
